acceptance/acceptance_test.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"github.com/cucumber/godog"
+)
+
+type (
+	inputFileName  struct{}
+	configFileName struct{}
+	testStateKey   struct{}
+)
+
+const (
+	policyInputFilename  = "input.json"
+	policyConfigFilename = "policy.json"
+)
+
+var (
+	//go:embed samples/policy-input-golden-container.json
+	sampleGCPolicyInput string
+)
```

Comment:
This could allow us to have more than one sample:

```suggestion
//go:embed samples/*.json
var samples embed.FS
```

Reply:
Oh the parenthesis were meant to help us group the samples together. I had another in there at some point. I'll remove it for now.

Reply:
https://github.com/enterprise-contract/ec-policies/pull/866